api/integrations/webhook/tests/test_views.py

```diff
@@ -0,0 +1,132 @@
+import json
+from unittest.case import TestCase
+
+import pytest
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from environments.models import Environment
+from integrations.webhook.models import WebhookConfiguration
+from organisations.models import Organisation, OrganisationRole
+from projects.models import Project
+from util.tests import Helper
+
+
+@pytest.mark.django_db
+class WebhookConfigurationTestCase(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        user = Helper.create_ffadminuser()
```

Comment:
Let's not use this old `Helper` class please. Why aren't we using the `pytest` and fixtures pattern here?